docs/process.md

```diff
@@ -77,35 +77,6 @@ How:
      commit it directly.
 
 
-Major version update (1.X.Y to 1.(X+1).0)
------------------------------------------
-
-When:
-
- * We should do such an update when we have a reasonable assurance of stability.
-
-Requirements:
-
- * All the requirements for a minor update.
- * No major change recently landed.
- * No major recent regressions have been filed.
- * All tests pass locally for the person doing the update, including the main
-   test suite (no params passed to `runner.py`), `other`, `browser`, `sockets`,
-   `sanity`, `binaryen*`. (Not all of those are run on all the bots.)
- * A minor version was recently tagged, no major bugs have been reported on it,
-   and nothing major landed since it did. (Bugs are often only found on tagged
-   versions, so a big feature should first be in a minor version update before
-   it is in a major one.)
-
-How:
-
-1. Follow all the steps for a minor version update.
-2. Merge the `incoming` branch to `master`. This should not be done immediately,
-   rather first we should at minimum see that CI and new builds are all green.
-   If a problem occurs, we may only merge to master the minor version update
-   that fixes things.
```

Comment:
Separate from this PR, perhaps we want to document when we update the major version number? (SemVer?)

Reply:
Done